state/factory/factory.go

```diff
@@ -427,7 +416,7 @@ func (sf *factory) initialize(ctx context.Context) error {
 		// not RunActionsCtx or no valid registry
 		return nil
 	}
-	ws, err := NewWorkingSet(sf.currentChainHeight, sf.dao, sf.rootHash(), sf.actionHandlers)
+	ws, err := newWorkingSet(sf.currentChainHeight, sf.dao, sf.rootHash(), sf.actionHandlers, sf.saveHistory)
```

Comment:
cannot find newWorkingSet

Reply:
in workingset.go
just rename to lower-case since it does not need to be exported